sunpy/physics/differential_rotation.py

```diff
@@ -322,3 +323,46 @@ def diffrot_map(smap, time=None, dt=None, pad=False, **diffrot_kwargs):
         out_meta['crval2'] = crval_rotated.Ty.value
 
     return sunpy.map.Map((out, out_meta))
+
+
+@u.quantity_input(xy=u.arcsec, dt='time')
+def rot_xy(xy, dt=None, time_start=None, time_end=None):
+    """
+    Utility function to apply solar differential rotation to solar XY
+    coordinates (helioprojective) given a time interval, or a starting
+    and end time for an observer on Earth.
+    Makes use of solar_rotate_coordinate.
+
+    Parameters
+    ----------
+    xy : array_like
+        Pair(s) of xy coordinates in arcsec. If more than one pair,
+        input should be an array_like with shape (2, N), where N is
+        the number of coordinate pairs.
+    dt : `~astropy.units.Quantity` or `datetime`, optional
```

Comment:
What is `datetime` here? Is it `datetime.datetime`? I'm just curious because the description says `dt` is an 'interval'.